### PR TITLE
Update zh_CN locale

### PR DESCRIPTION
**Pull request by yfdyh000**

### CODE REVIEW

File: extension/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "message": "选项…"
   },
   "customize_title": {
-    "message": "定制标题栏"
+    "message": "自定义标题栏"
```
Comment (whimboo) [inline extension/_locales/zh_CN/messages.json]: Just curious here what the difference is. When I translate both in Google the same translation is returned. Is it a different dialect?

**Comment by whimboo** [comment]
The changes look fine to me even the two existent ones are getting translated to English as the same content.

File: extension/_locales/zh_CN/messages.json (hunk 1 above)
Comment (yfdyh000) [reply extension/_locales/zh_CN/messages.json]: In my opinion, 定制 is customize / custom, 自定义 is customize / user-defined. Say it in another way, I think the former is the requirement / habit, the latter is to modify oneself.
In the previous translation, `"custom_template"` use the "自定义", I unify them to the better choice as I think.

Comment (whimboo) [reply extension/_locales/zh_CN/messages.json]: I see. Thank you for the explanation.

**Pull request merged**